uxarray/core/dataarray.py

```diff
@@ -1121,3 +1121,36 @@ def _slice_from_grid(self, sliced_grid):
             dims=self.dims,
             attrs=self.attrs,
         )
+
+    def constant_latitude_cross_section(self, lat: float, method="fast"):
```

Comment:
How about `cross_section_at_constant_latitude` or `cross_section_at_latitude`?

Reply:
Something analogous to our plotting and remapping accessors?

```Python
Grid.cross_section.constant_latitude()
Grid.cross_section.between_constant_latitudes()
Grid.cross_section.constant_longitude()
Grid.cross_section.between_constant_longitudes()
Grid.cross_section.gca_gca()
```

Reply:
Exactly, but I don't want to make it a priority for this PR as we can always design it over time and if it will require a long thought process

Reply:
Done.